Sources/KlaviyoUI/KlaviyoWebView/Resources/Scripts/consoleHandler.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WARNING This JS file should be ES5 compatible to support older versions of Android WebView
+ *
+ * Inject this script file from native code into the head of the HTML document
+ * Format string like this, where strWrapperScript is the contents of this file
+ * and strJsonConfig is the json-encoded handoff parameters detailed below
+ * Always be careful to put quotes around your string arguments! *
+ *
+ *    strWrapperScript + "('" + strJsonConfig + "');"
+ *
+ * @param strJsonConfig - JSON encoded handoff dictionary containing:
+ *    bridgeName: String - name of the native message handler
+ *    defaultAction: String - default action keyword for data posted from JS -> Native
+ */
+(function bridgeWrapper(strJsonConfig) {
+
+  //Initialize web wrapper object
+  window.WebViewBridge = new Bridge(JSON.parse(strJsonConfig));
+
+  /**
+   * Bridge object to handle communication between JavaScript and Native
+   */
+  function Bridge(opts) {
+    opts = opts || {};
+
+    return {
+      opts: opts,
+
+      /**
+       * Bool test if this is an Apple WKWebView
+       */
+      isApple: function () {
+        return !!window.webkit && !!window.webkit.messageHandlers[opts.bridgeName];
+      },
+
+      /**
+       * Method to post string message to native layer
+       *
+       * Native layer should implement a common interface
+       * for reacting to messages of particular type
+       *
+       * @param type {String}
+       * @param data {Object}
+       */
+      postMessageToNative: function (payload) {
+        try {
+          const serializedPayload = JSON.stringify(payload);
+
+          if (this.isApple()) {
+            window.webkit.messageHandlers[opts.bridgeName].postMessage(serializedPayload);
+          }
+        } catch (e) {
+          console.error("Failed to post message to native layer:", e.message);
```

Comment:
There is a potential to enter an endless loop, I had an updated version of this someplace that would undo the console-linkage below to prevent that, I'll find it

Reply:
```suggestion
          console.error("Failed to post message to native layer:", e.message);
          unlinkConsole()
```